### PR TITLE
Get gh issues

### PR DESCRIPTION
**Pull request by jonobrien**
still need to actually query backend for the routes

### CODE REVIEW

File: app/js/actions/issues.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+function removeIssueFailure(err) {
+  return {
+    type: REMOVE_ISSUE_FAILURE,
+    err,
+  }
+}
+
+export function addIssue(event) {
```
Comment (vuzDylan) [inline app/js/actions/issues.js]: event should be issue

File: app/js/components/HomePage.jsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import StockList from '../containers/StockList';
 import WeatherWidget from '../containers/WeatherWidget';
 import StockTicker from '../containers/StockTicker';
 import StockSearch from '../containers/StockSearch';
+import IssuesPage from '../containers/Issues';
```
Comment (vuzDylan) [inline app/js/components/HomePage.jsx]: Just call it Issues Page is reserved for router pages

File: app/js/containers/Issues.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  constructor() {
+    super();
+  }
+
+  componentDidMount() {
+    console.log("issues mounted");
+    this.props.dispatch(getIssues());
+  }
+
+  render(){
```
Comment (vuzDylan) [inline app/js/containers/Issues.jsx:23]: render should be somthing like 
const inssues = this.props.issues.map(issue => {
  return (
    // what ever an issue looks like
  )
})
return (
  div
    {issues}
  end div
)

**Comment by jonobrien** [comment]
mmk will fix

**Comment by jonobrien** [comment]
that html is bs'd cuz yolo

**Comment by jonobrien** [comment]
Somebody confirm the get actually works please :+1:

File: app/js/api/index.js [hunk 4]
```diff
@@ -15,6 +15,7 @@ class APIRoot {
     this.Buy = new API("stock_transactions/buy");
     this.Sell = new API("stock_transactions/sell");
     this.Stock = new API("stocks");
+    this.Issues = new API("api/github_issues");
```
Comment (kocsenc) [inline app/js/api/index.js]: So this probably only needs to remove ~~api~~; as you can see nothing else has that prefix.

File: app/js/containers/Issues.jsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  componentDidMount() {
+    this.props.dispatch(getIssues());
+  }
+
+  render(){
+    const allIssues = this.props.issues.map(issue => {
+      return (
+        <div>
+          <div className="ui attached header">{issue.title}</div>
+          <div className="ui attached segment">{issue.user.login}</div>
```
Comment (kocsenc) [inline app/js/containers/Issues.jsx]: You can confirm that `issue` has a `.user` and a `.login`?

File: app/js/reducers/issues.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+export default function issues(state = [], action) {
+  switch(action.type) {
+    case ADD_ISSUE_SUCCESS:
+    case GET_ISSUES_SUCCESS:
+    	return action.issues;
+    case REMOVE_ISSUE_SUCCESS:
+    default:
+      return state;
+  }
+}
```
Comment (kocsenc) [inline app/js/reducers/issues.js:16]: @vuzDylan this file look good?

File: app/js/api/index.js (hunk 4 above)
Comment (jonobrien) [inline app/js/api/index.js]: network tab showed client hitting incorrect route than what worked directly via browser hitting the api.

File: app/js/containers/Issues.jsx (hunk 5 above)
Comment (jonobrien) [inline app/js/containers/Issues.jsx]: according to the json returned it does

Comment (jonobrien) [inline app/js/containers/Issues.jsx]: or no

**Pull request merged**